### DISCUSSION TITLE
Questions for the default length of small intestinal in PKsim

### DISCUSSION THREAD

**Discussion by Won-ho** [Open-Systems-Pharmacology/Forum#1749] [Models]
Hello, PKsim users

I'm Wonho Kang working as a postdoc in the UF.
I have some questions about the default values of small intestinal length in PKSim. 

![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/52517358/52c613a3-7e09-4335-baf1-59068a66cda4)

1. There isn't any data origin provided in the table. 
2. Additionally, those values are not close to real lengths. Generally, the length of the jejunum and ileum are approximately 2.5 and 3.0 meters, respectively. 

If somebody knows why the default values are set like that, please let me know. 

Thank you in advance.

**Comment by Yuri05** [comment]
The values are based on the 
[**ICRP, 2002. Basic Anatomical and Physiological Data for Use in Radiological Protection Reference Values. ICRP Publication 89. Ann. ICRP 32 (3-4)**](https://www.icrp.org/publication.asp?id=ICRP%20Publication%2089)

You can find the reference values for the _default length of the small intestine and its segments for mean individual_ on page 114:
<img width="600" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/25061876/6b42ac33-b460-4c0e-a748-79d922460d0d">

>Additionally, those values are not close to real lengths. Generally, the length of the jejunum and ileum are approximately 2.5 and 3.0 meters, respectively.

You will find the explanation in the same report:

_(232) The anatomical length of the intestine represents length as measured at
autopsy or from material removed during surgery, while the physiological length
represents length as measured in a living person. **Anatomical lengths are often given in
standard textbooks of anatomy. These values are usually higher than the physiological
lengths, unless they were measured shortly after death when shortening of the intestines
may be observed.** Soon after death, the tissue loses its tonus, resulting in elongation._

_(233) **Reference values for the length of the small intestine given here refer to the
physiological length.** Reference values for newborns and adults are based on central
estimates of data summarised in Table 58 of Publication 23 (ICRP, 1975). For both
newborns and adults, the central estimates of the length of the small intestine are
approximately 1.6 body height. This relation is assumed to hold for ages 1–15 years
and for adult females._
